LilitYeghiazaryan/function_26

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <vector>
+#include <limits>
+#include <cmath>
+
+double f (int a, int b, int c)
+{
+    return std::cbrt(a * b * c);
+}
+
+double max_f(std::vector<int>& v, double (*f) (int, int, int))
+{
+    double max = std::numeric_limits<double>::min();
```

Comment:
For floating-point types with denormalization, min returns the minimum positive normalized value. Note that this behavior may be unexpected, especially when compared to the behavior of min for integral types. To find the value that has no values less than it, use numeric_limits::lowest.

Or -DBL_MAX